input/docs/cs/convolution.md

```diff
@@ -0,0 +1,47 @@
+title: Konvoluce - Klouzavý průměr
```

Comment:
o klouzavém průměru pak nikde není ani zmínka. Nějakou přidej (vysvětlit co to znamená, co to znamená ve vztahu ke konv.)